src/core/custom_tex_cache.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <unordered_map>
+#include <vector>
+#include "common/common_types.h"
+
+namespace Core {
+struct CustomTexInfo {
+    u32 width;
+    u32 height;
+    std::vector<u8> tex;
+};
+
+// TODO: think of a better name for this class...
+class CustomTexCache {
+public:
+    const bool IsTextureDumped(const u64 hash);
+    void SetTextureDumped(const u64 hash);
```

Comment:
```suggestion
    void SetTextureDumped(u64 hash);
```

the const can remain in the definition itself.